example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -0,0 +1,229 @@
+//
+// Example that demonstrates Offboard velocity control using LOCAL_NED & BODY_NED
+//
+// Author: Julian Oes <julian@oes.ch>, Shakthi Prashanth <shakthi.prashanth.m@intel.com>
+#include <iostream>
+#include <cmath>
+#include <thread>
+#include <chrono>
+#include <dronecore/dronecore.h>
+
+using namespace dronecore;
+
+#define ERROR_CONSOLE_TEXT "\033[31m" //Turn text on console red
+#define TELEMETRY_CONSOLE_TEXT "\033[34m" //Turn text on console blue
+#define NORMAL_CONSOLE_TEXT "\033[0m"  //Restore normal console colour
+
+void wait_msec(int msec)
+{
+    std::this_thread::sleep_for(std::chrono::milliseconds(msec));
+}
+
+void wait_sec(int sec)
+{
+    std::this_thread::sleep_for(std::chrono::seconds(sec));
+}
+
+int main(int argc, char **argv)
+{
+    DroneCore dc;
+    std::string offb_mode = (argc > 1) ? argv[1] : "";
+
+    if (argc == 1 || (offb_mode != "local-ned" && offb_mode != "body-ned")) {
```

Comment:
can we have this default to local? this would simplify the main routine, and perhaps use `"body"` instead of `"body-ned"` for the argument, just to simplify this

Reply:
ok.. Do you mean when the example is launched with no arguments, it should start offboard with local NED coordinates mode ?

Reply:
As of now, we're demonstrating both NED & Body co-ordinates. So, removed command-line handling.